### PR TITLE
Create Readme PT VERSION

### PR DESCRIPTION
**Pull request by anaisabelamr**

### CODE REVIEW

**Comment by spouliot** [comment]
@anaisabelamr thank you for your contribution :)
Could you sign the CLA agreement ?
We'll look at find someone, who know Portuguese, to review and find the correct file name for it.
Thanks again!

**Comment by mandel-macaque** [comment]
@viniciusjarina I think you can give us a hand with the language :)

File: Readme PT VERSION [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| master       | [![xamarin-macios-builds-master][1]][2]   |
+
+[1]: https://jenkins.mono-project.com/view/Xamarin.MaciOS/job/xamarin-macios-builds-master/badge/icon
+[2]: https://jenkins.mono-project.com/view/Xamarin.MaciOS/job/xamarin-macios-builds-master
+
+## Bem Vindo!
+
+Este módulo é o principal repositório para **Xamarin.iOS** e **Xamarin.Mac**.
+
+Esses SDKs nos permitem criar aplicativos para iOS, tvOS, watchOS e macOS utilizando os mesmos controles de interface do usuário que teríamos em Objective-C e Xcode, exceto com a flexibilidade e elegância de uma linguagem moderna (C #), o poder do .NET Base Class Library (BCL) e duas IDEs de primeira classe - Visual Studio para Mac e Visual Studio - na ponta dos dedos.
```
Comment (viniciusjarina) [inline Readme PT VERSION]: Utilizando o Xamarin.iOS e Xamarin.Mac é possível criar aprivativos para iOS, tvOS, watchOS e macOS utilizando os mesmos controles de interface e os mesmos frameworks nativos utilizados no Xcode. No entanto podendo usufruir das vantagens do .NET Framework e da linguagem C# com um ambiente integrado de desenvolvimento do Visual Studio e Visual Studio para Mac.

File: Readme PT VERSION [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+- [Submeta bugs e solicitações de recursos(requests)](https://github.com/xamarin/xamarin-macios/wiki/Submitting-Bugs-&-Suggestions)
+- [Revise as alterações do código-fonte](https://github.com/xamarin/xamarin-macios/pulls)
+- [Submeter pull requests](https://github.com/xamarin/xamarin-macios/wiki/How-to-Contribute#pull-requests) para resolver issues e consertar bugs.
+
+## Contribuindo
+
+Se você estiver interessado em corrigir problemas e contribuir diretamente para a base de código, consulte o documento [Como contribuir](https://github.com/xamarin/xamarin-macios/wiki/How-to-Contribute), que abrange o seguinte:
+
+- Como [rodar](https://github.com/xamarin/xamarin-macios/wiki/Build-&-Run) da fonte.
```
Comment (viniciusjarina) [inline Readme PT VERSION]: Como [executar](https://github.com/xamarin/xamarin-macios/wiki/Build-&-Run) a partir do código fonte.

File: Readme PT VERSION [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+[1]: https://jenkins.mono-project.com/view/Xamarin.MaciOS/job/xamarin-macios-builds-master/badge/icon
+[2]: https://jenkins.mono-project.com/view/Xamarin.MaciOS/job/xamarin-macios-builds-master
+
+## Bem Vindo!
+
+Este módulo é o principal repositório para **Xamarin.iOS** e **Xamarin.Mac**.
+
+Esses SDKs nos permitem criar aplicativos para iOS, tvOS, watchOS e macOS utilizando os mesmos controles de interface do usuário que teríamos em Objective-C e Xcode, exceto com a flexibilidade e elegância de uma linguagem moderna (C #), o poder do .NET Base Class Library (BCL) e duas IDEs de primeira classe - Visual Studio para Mac e Visual Studio - na ponta dos dedos.
+
+Este repositório é onde desenvolvemos os SDKs Xamarin.iOS e Xamarin.Mac. Existem algumas maneiras pelas quais você pode contribuir, por exemplo:
```
Comment (viniciusjarina) [inline Readme PT VERSION]: Neste repositório é desenvolvido os SDKs  Xamarin.iOS e Xamarin.Mac. Existem algumas maneiras pelas quais você pode contribuir, por exemplo:

**Comment by mandel-macaque** [comment]
@anaisabelamr could you sing the CLA so that we can merge this?

**Comment by rolfbjarne** [comment]
Unfortunately we can't accept this pull request until the CLA is signed.

I'm closing the pull request since it's been over three months since it was opened. Feel free to reopen once the CLA has been signed.

**Pull request closed without merging**